releases.md

```diff
@@ -36,13 +36,15 @@ Then check that all completed issues/pull requests are referenced (linked) from
 
 ## Release process of major versions
 
-* Ensure changelogs and update guides are up-to-date and complete.
+* Ensure issues of current GitHub project are either done or moved, changelogs and update guides are up-to-date and all tests run successfully.
 * Create a new section for the new version in the changelogs on master branch and move all unreleased entries to it.
 * Create the appropriate branch as described in the [Git](#git) section.
-* In each repository on the release branch, remove the `prerelease: Preview` from `docs/antora.yml`.
-* Tag the latest commit in each release branch with the specific version, e.g. `3.0`.
-* In each repository on the master branch, set `version` in `docs/antora.yml` to the next major version number (e.g. `3.x`).
-* In each quickstarter inside `ods-quickstarters` on the master branch, set `version` to the next major version number (e.g. `3.x`) in the `files/metadata.yml` file.
+* In each repository on the release branch, remove the `prerelease: Preview` from `docs/antora.yml` and replace the `## Unreleased` line in `CHANGELOG.md` with the specific version and date (e.g. `## [3.0] - 2020-08-11`).
```

Comment:
I think the part about the changelog should be aligned with the bullet point above...

Reply:
removed it